Day 1 extra.py

```diff
@@ -0,0 +1,37 @@
+# Read the variables from the CSV
+import csv
+
+
+
+file = csv.reader(open("/Users/jkelly2/Documents/santa.csv"))
+
+# Create the empty array to place the values into
+chronal_array = []
+
+# iterate over the file to place the values into the array
+for row in file:
+    chronal_array += row
```

Comment:
This shouldn't be working, have you confirmed that it does what you expect?
if row is a `str`, `list += row` should split `row` into a list, so `-2` would become `['-', '2']`

Reply:
Would make a good point for a unit test I guess !